src/syntax_mapping/builtins/linux/50-podman-quadlet.toml

```diff
@@ -0,0 +1,7 @@
+# see `man quadlet`
+[mappings]
+"INI" = [
+    "**/containers/systemd/*.{container,volume,network,kube,image}",
```

Comment:
This seems easy to create a regular syntax regression test for? That would be an excellent test for the fix I think.

Reply:
I think the argument is that the existence of this mapping alone is kind of a regression test for the `{…}`-based glob patterns. Of course, it would be nice to have syntax tests on top, but I'm personally fine with merging this as is.